### PR TITLE
Use dynamic namespace and the right roles perms in the Helm chart

### PR DESCRIPTION
**Pull request by luigibarbato**
Thanks to @ralgozino we have identified a number of issues within permission manager behavior regarding role creation in that the service account namespace is hardcoded on the backend side. In contrast, the role subjects namespace is hardcoded on the frontend side. Since this issue needs to be well analyzed and discussed, this PR implements a workaround to curb the problem in the short term.

In addition, we have identified that the default values of permissions assigned to different roles in the helm chart are not the correct ones.

### CODE REVIEW

**Review by ralgozino — CHANGES_REQUESTED** [review]
Please double-check the dynamic seeds, There are some permissions missing compared to the `deployments/kubernets/seeds/seed.yml` file.

File: web-client/src/components/Header.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function Header() {
     <nav className="flex items-center justify-between flex-wrap bg-teal-500 p-6">
       <div className="flex items-center flex-shrink-0 text-white mr-6">
         <span className="font-semibold text-xl tracking -tight">
-          Permissions Manager
+          Permission Manager
```
[the remark this answers is not in the document]
Comment (luigibarbato) [reply web-client/src/components/Header.tsx:9]: 🤣

**Review by ralgozino — APPROVED** [review]
LGTM 🎸

**Comment by luigibarbato** [comment]
> LGTM 🎸

💣

**Pull request merged**